### PR TITLE
[FIX] use of user_id fixed

### PR DESCRIPTION
**Pull request by JoelZilli**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/642).

File: crm_teams_ux/views/crm_lead_views.xml [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+        <field name="inherit_id" ref="crm.crm_lead_view_form"/>
+        <field name="arch" type="xml">
+            <field name="user_id" position="attributes">
+                <attribute name="domain">[('company_ids', 'in', user_company_ids)]</attribute>
+                <attribute name="groups">sale_ux.group_allow_any_user_as_salesman</attribute>
+            </field>
+            <field name="user_id" position="after">
+                <field name="user_id" groups="!sale_ux.group_allow_any_user_as_salesman"/>
+            </field>
+        </field>
```
Comment (nicomacr) [inline crm_teams_ux/views/crm_lead_views.xml]:
```suggestion
            <xpath expr="//field[@name='user_id'][1] position="attributes">
                <attribute name="domain">[('company_ids', 'in', user_company_ids)]</attribute>
                <attribute name="groups">sale_ux.group_allow_any_user_as_salesman</attribute>
            </field>
            <xpath expr="//field[@name='user_id'][2] position="attributes">
                <attribute name="domain">[('company_ids', 'in', user_company_ids)]</attribute>
                <attribute name="groups">sale_ux.group_allow_any_user_as_salesman</attribute>
            </field>
            <xpath expr="//field[@name='user_id'][1] position="after">
                <field name="user_id" groups="!sale_ux.group_allow_any_user_as_salesman" context="{'default_sales_team_id': team_id}" widget="many2one_avatar_user"/>
            </field>
            <xpath expr="//field[@name='user_id'][2] position="after">
                <field name="user_id" groups="!sale_ux.group_allow_any_user_as_salesman" context="{'default_sales_team_id': team_id}" widget="many2one_avatar_user"/>
            </field>
        </field>
```

**Review by nicomacr — CHANGES_REQUESTED** [review]
@JoelZilli please check this comment below

**Comment by nicomacr** [comment]
@roboadhoc r+

**Pull request closed without merging**